MeshGeoToolsLib/BoundaryElementsAtPoint.cpp

```diff
@@ -52,9 +52,11 @@ BoundaryElementsAtPoint::BoundaryElementsAtPoint(
     if (!multiple_nodes_allowed)
     {
         OGS_FATAL(
-            "BoundaryElementsAtPoint: the mesh node searcher found %d points "
-            "near the requested point (%f, %f, %f) in the mesh, while exactly "
-            "one is expected. Node  (id=%d) (%f, %f, %f) has distance %f.",
+            "BoundaryElementsAtPoint: the mesh node searcher found {:d} points "
+            "near the requested point ({:f}, {:f}, {:f}) in the mesh, while "
+            "exactly "
+            "one is expected. Node  (id={:d}) ({:f}, {:f}, {:f}) has distance "
+            "{:f}.",
```

Comment:
Changes could be better formatted.

Reply:
Manual work, unfortunately. Sometimes the breaks are for better readability of the strings, and should be kept as they are. So no automatic formatting possible.